RecoParticleFlow/PFProducer/python/pfLinker_cff.py

```diff
@@ -2,10 +2,10 @@
 
 import RecoParticleFlow.PFProducer.pfLinker_cfi
 import RecoParticleFlow.PFProducer.particleFlowTmpPtrs_cfi
-particleFlow = RecoParticleFlow.PFProducer.pfLinker_cfi.pfLinker.clone()
-particleFlow.PFCandidate = [cms.InputTag("particleFlowTmp")]
+particleFlowTmp2 = RecoParticleFlow.PFProducer.pfLinker_cfi.pfLinker.clone()
+particleFlowTmp2.PFCandidate = [cms.InputTag("particleFlowTmp")]
 particleFlowPtrs = RecoParticleFlow.PFProducer.particleFlowTmpPtrs_cfi.particleFlowTmpPtrs.clone()
-particleFlowPtrs.src = "particleFlow"
+particleFlowPtrs.src = "particleFlow"  # SRR 17-Dec-2019: 'particleFlow' will be post-puppi now. 
```

Comment:
I think that this should be moved downstream after ```particleFlow``` is known

Reply:
I don't think I understand this one. It's being imported by the particle flow sequence, so it doesn't need to be defined ahead of time, right?

Reply:
I mean that particleFlowPtrs should be defined in another file; it logically does not belong here anymore